client/python/lib/test/config.py

```diff
@@ -82,6 +50,23 @@ class ErrorCode(Enum):
     # DATA_LOSS = 15
     # DO_NOT_USE_RESERVED_FOR_FUTURE_EXPANSION_USE_DEFAULT_IN_SWITCH_INSTEAD = 20
 
+MODEL_SPEC_VALID = [
+    ("model_name", 1),
```

Comment:
Here this is not clear what the integer values mean, maybe add single line comment with field names?

Reply:
The same goes for all test case definitions